src/BuildIt.Forms/BuildIt.Forms.Controls/Camera.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BuildIt.Forms.Controls
+{
+    /// <summary>
+    /// Information about a camera device
+    /// </summary>
+    public class Camera : ICamera
+    {
+        /// <summary>
+        /// Gets the unique identifier for the camera
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Gets the direction the camera is facing
+        /// </summary>
+        public CameraPreviewControl.CameraFacing CameraFacing { get; set; }
```

Comment:
Can we pull the enum out so it isn't prefixed by the control

Reply:
done